docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc

```diff
@@ -0,0 +1,146 @@
+[[grant-access-to-elasticsearch]]
+= Grant standalone {agent}s access to {es}
+
+You can use either API keys or user credentials to grant standalone {agent}s
+access to {es} resources. The following minimal permissions are required to send
+logs, metrics, traces, and synthetics to {es}:
+
+* `monitor` cluster privilege
+* `auto_configure` and `create_doc` index privileges on `logs-*`, `metrics-*`,
+`traces-*`, and `synthetics-*`.
+
+It's recommended that you use API keys to avoid exposing usernames and passwords
+in configuration files.
+
+If you're using {fleet}, refer to
+{fleet-guide}/fleet-enrollment-tokens.html[{fleet} enrollment tokens].
+
+[discrete]
+[[create-api-key-standalone-agent]]
+== Create API keys for standalone agents
+
+NOTE: API keys are sent as plain-text, so they only provide security when used
+in combination with Transport Layer Security (TLS). Our
+{ess-product}[hosted {ess}] on {ecloud} provides secure, encrypted connections
+out of the box! For self-managed {es} clusters, refer to
+<<output-elasticsearch-pki-certs-authentication-settings>>.
+
+You can set API keys to expire at a certain time, and you can explicitly
+invalidate them. Any user with the `manage_api_key` or `manage_own_api_key`
+cluster privilege can create API keys.
+
+For security reasons, we recommend using a unique API key per {agent}. You
+can create as many API keys per user as necessary.
+
+To create an API key for {agent}:
+
+. In {kib}, navigate to *Stack Management > API keys* and click
+*Create API key*.
+
+. Enter a name for your API key and select *Restrict privileges*. In the role
+descriptors box, copy and paste the following JSON. This example creates an
+API key with privileges for ingesting logs, metrics, traces, and synthetics:
++
+[source,json]
+----
+{
+  "standalone_agent": {
+    "cluster": [
+      "monitor"
+    ],
+    "indices": [
+      {
+        "names": [
+          "logs-*", "metrics-*", "traces-*", "synthetics-*"
+        ],
+        "privileges": [
+          "auto_configure", "create_doc"
+        ]
+      }
+    ]
+  }
+}
+----
```

Comment:
Can we add a note that in many scenarios not all these permissions are needed. For example if not apm or synthetics is used, only logs and metrics are necessary.